sdk/purview/purview-workflow-rest/test/public/workflow.spec.ts

```diff
@@ -0,0 +1,115 @@
+          actions: {
+            Condition: {
+              type: "If",
+              actions: {
+                "Create glossary term": { type: "CreateTerm", runAfter: {} },
+                "Send email notification": {
+                  type: "EmailNotification",
+                  inputs: {
+                    parameters: {
+                      emailMessage:
+                        "Your request for Glossary Term @{triggerBody()['request']['term']['name']} is approved.",
+                      emailRecipients: ["@{triggerBody()['request']['requestor']}"],
+                      emailSubject: "Glossary Term Create - APPROVED",
+                    },
+                  },
+                  runAfter: { "Create glossary term": ["Succeeded"] },
+                },
+              },
+              else: {
+                actions: {
+                  "Send reject email notification": {
+                    type: "EmailNotification",
+                    inputs: {
+                      parameters: {
+                        emailMessage:
+                          "Your request for Glossary Term @{triggerBody()['request']['term']['name']} is rejected.",
+                        emailRecipients: ["@{triggerBody()['request']['requestor']}"],
+                        emailSubject: "Glossary Term Create - REJECTED",
+                      },
+                    },
+                    runAfter: {},
+                  },
+                },
+              },
+              expression: {
+                and: [
+                  {
+                    equals: [
+                      "@outputs('Start and wait for an approval')['body/outcome']",
+                      "Approved",
+                    ],
+                  },
+                ],
+              },
+              runAfter: { "Start and wait for an approval": ["Succeeded"] },
+            },
+            "Start and wait for an approval": {
+              type: "Approval",
+              inputs: {
+                parameters: {
+                  approvalType: "PendingOnAll",
+                  assignedTo: ["83ffc40f-f670-49b0-a17e-30c7f74f209a"],
+                  title: "Approval Request for Create Glossary Term",
+                },
+              },
+              runAfter: {},
+            },
+          },
+        },
+        isEnabled: true,
+        triggers: [
+          {
+            type: "when_term_creation_is_requested",
+            underGlossaryHierarchy: "/glossaries/5dae5e5b-5aa6-48f1-9e46-26fe7328de71",
+          },
+        ],
+      },
+    };
+    const createResult = await client.path("/workflows/{workflowId}", workflowId).put(options);
+
+    if (isUnexpected(createResult)) {
+      throw createResult.body.error;
+    }
+  });
+
+  it("should get a workflow", async function () {
+    const getResult = await client.path("/workflows/{workflowId}", workflowId).get();
+    if (isUnexpected(getResult)) {
+      throw getResult.body.error;
+    }
+  });
```

Comment:
For example here check that the status code is what you expect. And that some properties of the result.body are set as you expect

Reply:
Fixed, thanks